mirna_explorer/repository/create_database.py

```diff
@@ -28,6 +42,35 @@
     print(f'created database in {DB_PATH}')
     db_connection.commit()
 
-db_connection.close()
 
+# populate the seed_match_type table
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (0, '8mer'))
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (1, '7mer-m8'))
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (2, '7mer-A1'))
+
+# populate the mirnas table, here we consider only the human miRNAs
+all_mirnas = pd.read_csv(DATA_FOLDER + MIRNA_FAMILY_INFO_FILE, delimiter='\t')
+# 9606 is the species id for homo sapiens
+human_mirnas = all_mirnas.loc[all_mirnas['Species_ID'] == 9606, :]
+
+with open(DATA_FOLDER + MIRNA_ID_DICTIONARY_FILE) as infile:
+    content = csv.reader(infile, delimiter='\t', quoting=csv.QUOTE_NONE)
+    header = content.__next__()
+    for row in content:
+        mirna_family = row[0]
+        mirna_family = mirna_family
+        mirna_id = int(row[1])
+        sequence = human_mirnas.loc[human_mirnas['MiRBase_ID'] ==
+                                    mirna_family.replace('hsa-mir', 'hsa-miR'),
```

Comment:
What is this comparison? It compares a Series to a string, but I don't understand what it does. You can store the comparison in a variable. If there is a way to compare without string sensitivity, then it would be cleaner as we would avoid str.replace().